classes/publication/PublicationCategory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace PKP\publication;
+
+use Illuminate\Database\Eloquent\Model;
+
+class PublicationCategory extends Model
+{
+    protected $table = 'publication_categories';
+    protected $primaryKey = 'publication_category_id';
+    public $timestamps = false;
+
+
+    protected $fillable = [
+        'publication_id', 'category_id'
+    ];
+
+    /**
+     * Get the categories associated with a publication.
+     *
+     */
+    public static function getCategoriesByPublicationId(int $publicationId)
+    {
+        return self::where('publication_id', $publicationId)->pluck('category_id');
+    }
+
+    /**
+     * Assign categories to a publication.
+     *
+     */
+    public static function assignCategoriesToPublication(int $publicationId, array $categoryIds)
```

Comment:
This could go into a `Repository` class. [Example](https://github.com/pkp/pkp-lib/blob/main/classes/stageAssignment/Repository.php)

Reply:
Also, don't forget return typehints, even if it's just `void` for functions that don't return values.